### PR TITLE
Fix path normalization

### LINKED ISSUE #2: Option to start from project root
Starting from project root rather than the driver root would be great.

### PR DESCRIPTION
**Pull request by innerlee**
Fix #2 .
The cause is,
1. `/` v.s. `\`
2. leading `/` in windows
bug

edit:
review the third commit is okay. The first two are just formatting :P

### CODE REVIEW

File: src/extension.js [hunk 1]
```diff
@@ -23,8 +23,12 @@ function _isDirectory(file) {
 function createBreadCrumbItemsFromFile(fileName, callback) {
     // this wall of code full of shit but do exactly what it should
     // no power to refactor it
+    fileName = path.normalize(fileName)
```
Comment (danields761) [inline src/extension.js]: Can u please add semicolon at the end of new lines (26, 28, 44)?
It'll make this PR same formatted with rest of code)

Comment (innerlee) [reply src/extension.js]: Sure!

**Comment by innerlee** [comment]
Just found some bug in implementation!

edit:
the leading `/` in `fileName` will cause trouble for the method `path.relative`

will fix

**Comment by innerlee** [comment]
With the fix, it still fails in two cases:
1. multi-root workspace
2. no workspace

will fix these two soon

**Comment by innerlee** [comment]
all fixed. pls test

**Comment by danields761** [comment]
Looks good onto Ubuntu, at least nothing crashes :D Multi-root workspace works great
But i haven't any possibility to test this fix on Windows %) Did you tested it?

**Comment by innerlee** [comment]
Tested on win10

**Comment by innerlee** [comment]
Comments?

**Comment by danields761** [comment]
All work nice, PR almost ready to be merged
Please, add yourself in contributors section into `package.json`
Thx!

**Comment by innerlee** [comment]
Sorry cannot find the section. Could you help edit the file?

**Comment by danields761** [comment]
Please, see this [link](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) :)

**Comment by danields761** [comment]
Thank you!
Shortly will make new release :)

**Comment by innerlee** [comment]
That's great!

**Pull request merged**